### ISSUE TITLE
allow admins to start a vote without voting

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#1940]
Currently, when one starts a vote, it is considered this person if in favor for the change. No way to be neutral.

**Comment by Gireen** [comment]
why only admins? this could be done by default for everyone

**Comment by necessarily-equal** [comment]
This is one of the case where both options feel equally good to me, and could equally get a bug report to change the default.

### Pros for voting by default
* Simpler
* (IMO) less confusing. I think most of the time if you ask for (e.g) a draw, you actually want a draw. Or you wouldn't go into the effort of asking it.
* It has the advantage of not making you wonder why the vote isn't working when you are alone, while the thing that actually happened is that no vote has been cast yet.

### Pros for not voting by default
* More neutral, allows for pooling instead of suggesting it has to be done.
* ???

I think I like the status quo better.

**Comment by bmorel** [comment]
> Pros for not voting by default

Also note that if it was possible to erase or change one own's vote, this would close this ticket imo.

**Comment by bmorel** [comment]
@necessarily-equal  one con that I see is: there is no obvious clue you can/need to vote for or against something. THere's also no way to just say "don't care".

Do you @all think it would be a good idea to "center print" the vote suggestion, with "F1: yes, F2: no, F3: don't care" ?

**Comment by DolceTriade** [comment]
I really do not want to have an abstain option. I think CP isn't the best option because
 a) It's temporary
 b) it can be overridden by another CP. 

Perhaps a better way to handle votes is to give votes a background and do something with the vote menu to attract attention until one has voted. Then the background can go away and we can decrease the opacity or something.

**Comment by necessarily-equal** [comment]
Note that if we start making votes anouncement big and shiny, we risk having people complaining they lost because of that. I've seen such complaints already.

This question is another one anyway, so maybe let's go back to the question at hand. I'm ok with the idea of being able to change one's own vote, but I also dislike the idea of adding a third key to vote (abstain/neutral) because it changes the key semantics (currently we have F1/F2 and F3/F4 so it would have to be changed to F1/F2/F3 and F4/F5/F6, and F4 which used to mean "no" would now mean "yes"). I'm also unsure that we can update the presets flawlessly when the are overriden.

Maybe if you voted F1 you could just press that again to retract your vote ?

**Comment by DolceTriade** [comment]
I'm ok with changing the vote though.

**Comment by bmorel** [comment]
I agree that a "no opinion" vote would be bad. Better to be able to cancel a vote by "over-voting" the choice.

**Comment by DolceTriade** [comment]
Agree with allowing you to change your vote exactly once. Don't want to let players just press F1 and F2 the entire duration of the vote.

**Comment by necessarily-equal** [comment]
Maybe allowing to change it twice (allowing to go back to your original choice) would be better though.

**Comment by bmorel** [comment]
Just let the admin decide that kind of thing... number of cancels votes and cooldown before each.

**Comment by DolceTriade** [comment]
Sure, a cvar for configuration sounds reasonable. Seems like the general consensus for this topic is:

 - You cannot start a vote without already voting yes.
 - You **can** change your vote if you have already voted.

Seem reasonable?

**Comment by necessarily-equal** [comment]
I really don't like the idea of making number of votes configurable on 
the admin, it would make it really confusing for the user. You really 
don't want the user to be confused. Making votes change limits 
disable-able by an admin may be ok though, because it doesn't lock the 
user in a vote they didn't want to take.

**Comment by bmorel** [comment]
I don't see why it would confuse users. Just have a message showing the number of vote changes the user still have and there's no more problems.

**Comment by bmorel** [comment]
I implemented a way to change vote in #2597, but I won't implement a way to limit number of those instances, because:

1. I don't think it's useful
2. the vote code is really convoluted (but a bit less after my PR will be merged)
